orbit/cmd/orbit/signal_unix.go

```diff
@@ -0,0 +1,13 @@
+package main
```

Comment:
here and below, are we missing the comment with the build tags?

Reply:
Great catch (I thought `_unix.go` would work, but looks like it work in the build tag only: https://tip.golang.org/doc/go1.19#go-unix)

Reply:
nice! I'm guessing the CI check that builds Orbit will eventually catch this, but to save you some time:

```
~/fleet $ GOOS=windows GOARCH=amd64 go build -o orbit-windows.exe ./orbit/cmd/orbit
# github.com/fleetdm/fleet/v4/orbit/cmd/orbit
orbit/cmd/orbit/signal_windows.go:6:2: imported and not used: "syscall"
```

---

unrelated, I don't understand why we don't need the analogous flag comment in `signal_windows.go`, is the compiler that smart?

Reply:
(I mean, it works so LGTM, but I don't understand why!)